study/test/src/org/labkey/test/tests/study/StudyPublishTest.java

```diff
@@ -220,6 +220,8 @@ protected void doCreateSteps()
 
         setUnshiftedDateField(DATE_SHIFT_DATASET, UNSHIFTED_DATE_FIELD.getKey());
 
+        goToProjectHome();  // the prior operation leaves the test on the query metadata editor, scrolled down

```

Comment:
Can scroll instead of adding a navigation.
```suggestion
        scrollToTop();  // the prior operation leaves the test on the query metadata editor, scrolled down

```